composer.json

```diff
@@ -18,7 +19,7 @@
     ],
     "require": {
         "php": ">=5.3.3.",
-        "zendframework/zendframework": "~2.2"
+        "zendframework/zendframework": "~2.2 || ~3.0"
```

Comment:
Could you list exact required zend framework components instead of requiring the whole framework?